### ISSUE TITLE
Could not parse the Youtube page for URL (i.e. Most if not all Music Videos)

### ISSUE THREAD

**Issue by GMMcMahon** [flagbug/YoutubeExtractor#274]
**Windows 10 Pro Version 1703 Build 1506.483**

**WinTube for Youtube** in the Windows 10 Store.
by **Two Guys and an App LLC**

Could not parse the Youtube page for URL https://www.youtube.com/watch?v=n-D1EB74Ckg
This may be due to a change of the Youtube page structure.

**NOTE:** This error message comes up for most if not all of the music videos in the WinTube Program. All videos play fine on the Youtube website.

Please fix.

**Comment by hrkrx** [comment]
Yes, i can confirm this, for almost every musicvideo i get a 403.

EDIT:
this is only the case when using the nuget package, the actual source works. Even music videos with decrypt=true
